x/participation/types/genesis_test.go

```diff
@@ -83,6 +83,30 @@ func TestGenesisState_Validate(t *testing.T) {
 			},
 			valid: false,
 		},
+		{
```

Comment:
I think we should also have a valid case where the sum is equal. For default we have `0 == 0`